src/internal/event/EventInteractor.ts

```diff
@@ -169,6 +169,17 @@ export class EventInteractor {
   }
 
   trackFailure(apiId: ApiId, featureTag: string, error: BKTException): void {
+
+    if (error.type === MetricsEventType.UnauthorizedError) {
+      console.log('An unauthorized error occurred. Please check your API Key.')
```

Comment:
How about using the error level?

```suggestion
      console.error('An unauthorized error occurred. Please check your API Key.')
```